blocks/subscriptions-block/components/SubscriptionDialog/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_STYLES,
+  BUTTON_TYPES,
+  PrimaryFont,
+} from '@wpmedia/shared-styles';
+
+import './styles.scss';
+
+const SubscriptionDialog = ({
+  actionText,
+  actionUrl,
+  reasonPromptText,
+  headlineText,
+  linkText,
+  linkPromptText,
+  linkUrl,
+  subHeadlineText,
+}) => (
+  <PrimaryFont as="div" className="xpmedia-subscription-dialog">
+    {reasonPromptText
+      ? (
+        <div
+          className="xpmedia-subscription-dialog-reason-prompt"
+          dangerouslySetInnerHTML={{ __html: reasonPromptText }}
+        />
+      )
+      : null}
+    <div className="xpmedia-subscription-dialog-link-prompt">
+      {linkPromptText
+        ? (
+          <span
+            className="xpmedia-subscription-dialog-link-prompt-pre-link"
+            dangerouslySetInnerHTML={{ __html: linkPromptText }}
```

Comment:
is this text or should we say it could be any content? if it says text or string, I might expect no markup? naming question mostly

Reply:
`PropTypes.node`?

Reply:
Updated some of these to use PropTypes.oneOfType which is probably better in this case.